OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -388,6 +390,10 @@ abstract class MovePart : Activity
 
 				IsInterruptible = false; // See comments in Move.Cancel()
 
+				TurnsWhileMoving = move.mobile.Info.TurnsWhileMoving;
+				if (TurnsWhileMoving)
```

Comment:
```suggestion
				ToFacing = TurnsWhileMoving ? (To - From).Yaw + Move.ActorFacingModifier :  toFacing;
```
ToFacing should be reused like this

Reply:
I suggest store `(To - From).Yaw` into a var, the calculation of Yaw consist of a loop in WVec.cs, line 146 to get the best value, which maybe costly.

Reply:
this is storing it. Though yeah it would be better to just pass this as a `toFacing` parameter when instantiating MoveHaff

Reply:
Just get what you mean. Reused.